pydantic_ai_slim/pydantic_ai/_thinking_part.py

```diff
@@ -2,11 +2,10 @@
 
 from pydantic_ai.messages import TextPart, ThinkingPart
 
-START_THINK_TAG = '<think>'
-END_THINK_TAG = '</think>'
 
-
-def split_content_into_text_and_thinking(content: str) -> list[ThinkingPart | TextPart]:
+def split_content_into_text_and_thinking(
+    thinking_start_tag: str, thinking_end_tag: str, content: str
```

Comment:
We can pass a `tuple[str, str]` here as well, and I'd prefer to have that argument _after_ the `content`